Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -340,6 +340,8 @@ confirmRename=Are you sure you want to rename this item?
 contentLoeschen=Delete content
 copy=copy
 correctionK=C
+correctionReportProblem=Problem for task correction couldn't be reported!
```

Comment:
Please use something like `Correction issue message to previous task could not be reported!`.